calplus/conf/providers.py

```diff
@@ -29,23 +29,25 @@
                default=''),
     cfg.StrOpt('os_novaclient_version',
                default='2.1'),
```

Comment:
this should be changed along with your swift client version, too.
Maybe we should support Newton or Ocata?